src/main/java/com/github/manolo8/darkbot/Bot.java

```diff
@@ -32,21 +43,83 @@ public static void main(String[] args) throws IOException {
         LibSetup.setupLibraries();
         StartupParams params = new StartupParams(args);
 
-        checkJavaVersion(params);
         SwingUtilities.invokeLater(() -> new Main(params));
     }
 
-    private static void checkJavaVersion(StartupParams params) {
+    static void checkUniqueInstance(StartupParams params) {
```

Comment:
Keep order consistent, checkJavaVersion was here first, is called first, and should be called first, so keep that as the first of the 2 methods in h ere